src/main/resources/default/assets/tycho/scripts/charts.js.pasta

```diff
@@ -345,3 +345,28 @@ function barChart(selector, labels, datasets) {
         options: customConfig
     });
 }
+
+/**@
+ * Renders a doughnut chart.
+ * @param selector the selector to determine the target canvas. Also, the canvas can be passed in as DOM element
+ * @param labels an array of labels to use for the chart
+ * @param data array of numeric values to use for the chart
+ */
+function dougnutChart(selector, labels, data) {
+    const _canvas = sirius.querySelector(selector);
+
+    new Chart(_canvas, {
+        data: {
+            datasets: [{
+                data: data,
+                backgroundColor: SIRIUS_CHART_COLOR_WHEEL
```

Comment:
shouldn't we create a custom array here using pickFromColorWheel() n times? This would result in a custom chart per call (which is still constant in its colors)...